CHANGELOG.asciidoc

```diff
@@ -14,8 +14,7 @@ https://github.com/elastic/beats/compare/v7.4.1...v7.5.0[View commits]
 - Libbeat HTTP's Server can listen to a unix socket using the `unix:///tmp/hello.sock` syntax. {pull}13655[13655]
```

Comment:
If everything in this list is a breaking change, we need to do a better job of explaining how users are impacted and what they need to do to address the problem. It's not clear to me even after skimming the PR why these are considered breaking changes.